### PR TITLE
Enable no-response bot to auto close inactive issues

### PR DESCRIPTION
**Pull request by testforstephen**
Signed-off-by: Jinbo Wang <jinbwan@microsoft.com>

It auto closes those inactive issues:
- Marked as `"need info"`
- Inactive for 14 days

### CODE REVIEW

File: .github/workflows/no-response.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: No Response
+
+# **What it does**: Closes issues where the original author doesn't respond to a request for information.
+# **Why we have it**: To remove the need for maintainers to remember to check back on issues periodically to see if contributors have responded.
+# **Who does it impact**: Everyone that works on docs or docs-internal.
```
Comment (fbricon) [inline .github/workflows/no-response.yml:5]: that last line is irrelevant

File: .github/workflows/no-response.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    types: [created]
+  schedule:
+    # every morning at 00:00 am
+    - cron: '0 0 * * *'
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
```
Comment (fbricon) [inline .github/workflows/no-response.yml:18]: please use the SHA1 commit of that version, as per https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

**Comment by fbricon** [comment]
I think we need to manually go over all of https://github.com/redhat-developer/vscode-java/issues?q=is%3Aopen+is%3Aissue+label%3A%22need+info%22 to check if users actually gave some info, and remove the label if needed, before we apply this action.

**Comment by rgrunber** [comment]
I've gone over all the `need info` labels. I definitely learned some interesting things along the way and for the most part I've either :

- Removed `need info` because additional info was given, or I felt there was enough info that an investigation at a later time was worth it
- Closed the issue because it was either a duplicate, or it was very similar to issues we fixed in the past, etc.
- Left the issue because it had `need info`, no response / no meaningful information to make any progress

I've also bumped the "days inactive" for auto-close to 30, as 14 seemed a bit too short.

**Pull request merged**